src/async/imap/MCIMAPOperation.cpp

```diff
@@ -110,14 +110,18 @@ void IMAPOperation::start()
         IMAPAsyncConnection * connection = mMainSession->sessionForFolder(mFolder, mUrgent);
         setSession(connection);
     }
-    mSession->runOperation(this);
+    session()->runOperation(this);
 }
 
+namespace {
```

Comment:
I know that is a good best practice in C++, though could you remove anyway that anonymous namespace anyway?